tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "noImplicitAny": true,
     "allowSyntheticDefaultImports": false,
     "removeComments": true,
-    "noUnusedLocals": true,
+    "noUnusedLocals": false,
```

Comment:
Why disable the check for unused locals? 
What unused locals are in the project?

Reply:
I was experimenting with a __mocks__ for the twilio client that had a constructor that had props that were not used.
This is no longer the case, so i'll re-enable it